openhtf/io/station_api.py

```diff
@@ -699,7 +698,8 @@ def wait_for_update(self, test_uid, remote_state_dict, timeout_s):
           'RPC:wait_for_update() -> short-circuited wait (local was blank)')
       return self._serialize_state_dict(state._asdict())
 
-    state_dict_summary = self._summary_for_state(state)
+    state_dict, event = state.asdict_with_event()
```

Comment:
while you're here, maybe rename `event` to `update_event`, it's a big vague as-is.